.NET/Microsoft.Recognizers.Definitions.Common/French/DateTimeDefinitions.cs

```diff
@@ -24,6 +24,7 @@ public static class DateTimeDefinitions
 		public const string TillRegex = @"(?<till>au|[aà]|et|jusqu'[aà]|avant|--|-|—|——)";
 		public const string RangeConnectorRegex = @"(?<and>et|de la|au|[aà]|et\s*la|--|-|—|——)";
 		public const string RelativeRegex = @"(?<order>prochain|prochaine|de|du|ce|cette|l[ae]|derni[eè]re|pr[eé]c[eé]dente|au\s+cours+(de|du\s*))";
+		public const string StrictRelativeRegex = @"(?<order>prochain|prochaine|ce|cette|derni[eè]re|pr[eé]c[eé]dente|au\s+cours+(de|du\s*))";
```

Comment:
Why declare a new Regex but it's not used anywhere? Shouldn't other pieces reference it?